tracks/bash/exercises/grains/mentoring.md

```diff
@@ -39,15 +37,13 @@ like that:
 function main() {
   square="$1"
 
-  if [[ "$square" == "total" ]]; then
+  if [[ "$square" == 'total' ]]; then
```

Comment:
Note: The use of double quotes is perfectly fine here and somewhat more consistent. Either way is fine but I'm not sure there's a good argument in favor of making this change.

Reply:
As I know it's better to use single quotes where you don't need variables expansions

Reply:
I'm unaware of any guidance to that effect :) If you're aware of anything like that in any style guides, I'd love to see it.

When strings contain special/meta chars, then single quotes are quite useful. For regular strings, less so.

Reply:
I checked once again code styles and articles and didn't find anything except few examples in [Google code style](https://google.github.io/styleguide/shellguide.html#quoting), but they're inconsistent (few first examples use single quotes, other double quotes). But it's easier for me to read literal string when I saw  single quotes and I know that there are no expansions at all. Seems that this is taste question:) Thanks for pointing out.